docs/reference/compatibility.txt

```diff
@@ -168,6 +168,9 @@ Please refer to `the driver compatibility page
 <https://docs.mongodb.com/ruby-driver/current/reference/driver-compatibility/>`_
 for driver compatibility matrices.
 
+"D" in a column means support for that MongoDB server version is deprecated
+and will be removed in a next version.
```

Comment:
I think you should move this explanation up a section since "Ruby Compatibility" also uses this convention.

Reply:
Oh nvmd it is there already.